hack/mkpjpod.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Create a pod from a Prow job using the test-infra mkpj and mkpod utilities.
+# Requires the following environment variables (see the pj_env.py script for a
+# way to set them automatically):
+#
+# - BASE_REF
+# - BASE_SHA
+# - PULL_NUMBER
+# - PULL_SHA
+# - PULL_AUTHOR
+set -euo pipefail
+
+if [[ "$#" -ne 1 ]]; then
+    echo >&2 "Usage: $0 job_name"
+    exit 1
+fi
+job=$1
+img=registry.svc.ci.openshift.org/ci/test-infra:binaries
+podman run --rm -iv "$PWD:/tmp/release:Z" -w /tmp/release "$img" bash <<-EOF
```

Comment:
Hm are we that confident everyone is running `podman` already?

Reply:
I'm not :)

Reply:
Also the `:Z` should not be necessary, `:z` should suffice

Reply:
> Hm are we that confident everyone is running podman already?

Can we make them? =)

> I'm not :)

Shame on you.

> Also the :Z should not be necessary, :z should suffice

I always think of it as `z` == "security", `Z` == "more security". I'll have to find a better mnemonic (or RTFM).